docs/guides/events-list.rst

```diff
@@ -130,9 +130,13 @@ Entity events
 
 **update, <entity type>**
     Triggered before an update for the user, group, object, and site entities. Return false to prevent update.
+    The entity method ``getOriginalAttributes()`` can be used to identify which attributes have changed since
+    the entity was loaded.
```

Comment:
I will give it another look on Sunday. I will pretend it's Sabbath tomorrow.
On Dec 5, 2015 12:13 AM, "Steve Clay" <notifications@github.com> wrote:

> In docs/guides/events-list.rst
> <https://github.com/Elgg/Elgg/pull/9188#discussion_r46744136>:
>
> > @@ -130,9 +130,13 @@ Entity events
> >
> >  **update, <entity type>**
> >      Triggered before an update for the user, group, object, and site entities. Return false to prevent update.
> > +    The entity method ``getOriginalAttributes()`` can be used to identify which attributes have changed since
> > +    the entity was loaded.
>
> @hypeJunction <https://github.com/hypeJunction> I just made these both
> simpler.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/9188/files#r46744136>.
>